src/utils/sortableComponent.ts

```diff
@@ -0,0 +1,77 @@
+import Sortable from "sortablejs";
+const sortableComponentSet = new Set<SortableComponent>();
+const inactiveSortableComponentSet = new WeakSet<SortableComponent>();
+
+/**
+ * Defines interface for components with sorting functionality.
+ */
+export interface SortableComponent {
+  /**
+   * The Sortable instance.
+   */
+  sortable: Sortable;
+}
+
+/**
+ * Helper to keep track of a SortableComponent. This should be called in the `connectedCallback` lifecycle method as well as any other method necessary to rebuild the sortable instance.
+ *
+ * @param {SortableComponent} component - The sortable component.
+ * @param {HTMLElement} element - Any variety of HTMLElement.
+ * @param {SortableComponent} [options] - Sortable options object.
+ */
+export function connectSortableComponent(
+  component: SortableComponent,
+  element: HTMLElement,
```

Comment:
Do we have any use cases where `element` is different than the component's host? If not, using `component.el` could help simplify the util a bit until we need to support a different root sortable element.